mono/mini/Makefile.am.in

```diff
@@ -34,6 +34,52 @@ sgen_libs = \
 	$(monodir)/mono/utils/libmonoutils.la \
 	$(glib_libs)
 
+if ENABLE_LLVM
+# We need to define
+#    LLVM_CFLAGS
+#    LLVM_CXXFLAGS
+#    LLVM_LIBS
+#    LLVM_LDFLAGS
+
+if MONO_INTERNAL_LLVM 
+llvm_config=$(monodir)/llvm/usr/bin/llvm-config
+else
+llvm_config=$(EXTERNAL_LLVM_CONFIG)
+endif
+llvm_system_libs=$(shell $(llvm_config) --system-libs 2>/dev/null | grep -- -)
+LLVM_VERSION=$(shell $(llvm_config) --version)
+
+llvm_api_version_from_config=$(shell $(llvm_config) --mono-api-version 2>/dev/null)
```

Comment:
`llvm_api_version_from_config:=$(shell $(llvm_config) --mono-api-version 2>/dev/null)` otherwise you end up invoking the shell everytime you use the variable.